### PR TITLE
@@task_report-view supports task lookup by the ressource-id through the `tasks` parameter.

### PR DESCRIPTION
**Pull request by elioschmutz**
Pre-discussed with @deiferni : https://4teamwork.slack.com/archives/C01BHEGBVU1/p1603707985016500

Jira: https://4teamwork.atlassian.net/browse/NE-56

----- 

The `@@task_report` view is a folder-action which should also work for the gever-ui.

Currently, the view wants a list of `task_ids` which should be included in the report. Unfortunately, we don't have `task_ids` in the `@listing` endpoint of the frontend. Including it would mean, that we have to index the `task_ids` in solr and expose it int the `@listing` endpoint. This was not desired. Thus, we need another way to lookup the tasks.

Because the tasks in a default listing are always within the current admin unit, we can do a task lookup by the resource-id (`@id`)

This PR extends the `@@task_report` view with a new parameter: `tasks`. It provides a list of task resource ids which should be included in the report.

The view now provides two ways to include tasks:

by `task_id`:
`@@task_report?task_ids:list=1&task_ids:list=2`

by the resource id:
`@@task_report?tasks:list=https://example.com/ordnungssystem/dossier-1/task-1&tasks:list=https://example.com/ordnungssystem/dossier-1/task-2`

## Checklist (Must have)

_Everything has to be done/checked. Checked but not present means the author deemed it unnecessary._

- [x] Changelog entry
- [x] Link to issue (Jira or GitHub) and backlink in issue (Jira)

## Checklist (if applicable)

_Only applicable should be left and checked._

- API change:
  - [ ] Documentation is updated
  - If breaking:
    - [ ] api-change label added
    - [ ] #delivery channel notified about breaking change
    - [ ] Scrum master is informed
- New functionality:
  - [ ] for `document` also works for `mail`
  - [ ] for `task` also works for `forwarding`
- Upgrade steps (changes in profile):
  - [ ] Make it deferrable if possible
  - [ ] Execute as much as possible conditionally
- DB-Schema migration
  - [ ] All changes on a model (columns, etc) are included in a DB-schema migration.
  - [ ] Constraint names are shorter than 30 characters (`Oracle`)
- [ ] Change could impact client installations, client policies need to be adapted
- New translations
  - [ ] All msg-strings are unicode
- Change in schema definition:
  - [ ] If `missing_value` is specified, then `default` has to be set to the same value

### CODE REVIEW

**Review by njohner — APPROVED** [review]
LGTM, I'm just a bit unsure about using absolute URLs.

Well I just went through your discussion with @deiferni and it seems using `path` or `@id` was discussed already. I think I'd have argued for using `path`s, but as long as it's a conscious decision to introduce what I'd call inconsistencies, but it seems in that case because it's the direction we want to push towards in the future, well then fine with me.

**Comment by elioschmutz** [comment]
@njohner I don't know the "right" way. But i.e. the copy-move endpoint can handle absolute urls, paths or UID (https://github.com/plone/plone.restapi/blob/master/src/plone/restapi/services/copymove/copymove.py#L26). Perhaps we should also use something like the `get_object` method of `copymove` to provide this tree ways to lookup an object.

But for now, we just have to extend the `task_report` view to also provide a way to lookup tasks through a path or ID.

@deiferni could you summarize why you prefered to use `@id` instead of the `path`? Do you still agree with this decision?

**Comment by deiferni** [comment]
With the current implementation honestly i do not particularly care 😅 . 
In the thread i was arguing what might be a good idea with a service endpoint. What we are actually doing here is pass a list of references, and there are several ways this can be done with the current `plone.restapi` implementation. But we will have to touch this again eventually when we remove the browser views and switch to a service endpoint. So if you two prefer `path` i don't have any objection!

**Comment by elioschmutz** [comment]
Thanks @deiferni . I prefer to merge this PR 😂
